tools/autograph-monitor/monitor.go

```diff
@@ -67,37 +62,27 @@ func main() {
 	// prod or autograph dev code signing PKI roots and CA root
 	// certs defined in constants.go
 	conf.env = os.Getenv("AUTOGRAPH_ENV")
+	var err error
 	switch conf.env {
 	case "stage":
-		conf.rootHash = firefoxPkiStageRootHash
-		conf.truststore = x509.NewCertPool()
-		conf.truststore.AppendCertsFromPEM([]byte(firefoxPkiStageRoot))
-		conf.contentSignatureRootHash = firefoxPkiContentSignatureStageRootHash
-		conf.contentSignatureTruststore = x509.NewCertPool()
-		conf.contentSignatureTruststore.AppendCertsFromPEM([]byte(firefoxPkiContentSignatureStageRoot))
-		conf.depRootHash = ""
-		conf.depTruststore = nil
+		conf.truststore, conf.rootHashes, err = LoadCertsToTruststore(firefoxPkiStageRoots)
 	case "prod":
-		conf.rootHash = firefoxPkiProdRootHash
-		conf.truststore = x509.NewCertPool()
-		conf.truststore.AppendCertsFromPEM([]byte(firefoxPkiProdRoot))
-		conf.contentSignatureRootHash = firefoxPkiProdRootHash
-		conf.contentSignatureTruststore = conf.truststore
-		conf.depRootHash = firefoxPkiStageRootHash
-		conf.depTruststore = x509.NewCertPool()
-		conf.depTruststore.AppendCertsFromPEM([]byte(firefoxPkiStageRoot))
+		conf.truststore, conf.rootHashes, err = LoadCertsToTruststore(firefoxPkiProdRoots)
+		if err != nil {
+			break
```

Comment:
I'll admit I'm not stoked by having to reuse `err` across the switch (yes, i didn't mention this on the zoom; it's not a huge deal), but since we do, we probalby want to make this err and the `stage` one after it more specific with a 

`err = fmt.Errorf("unable to load production roots: %w", err)`

Reply:
Ah yeah, that could get confusing. I think I can make this a bit more explicit since we have 2 trust stores, why not have 2 errors?